### PR TITLE
Add useSelectedChain hook

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v3/src/hooks/useSelectedChain.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { bind } from '@react-rxjs/core';
+import { createSignal } from '@react-rxjs/utils';
+
+export const [selectedChain$, setSelectedChain] = createSignal<string | null>();
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/hooks/useSelectedChain.ts:4]: do we need to export `selectedChain$`?

File: packages/tempus-client_v3/src/hooks/useSelectedChain.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { bind } from '@react-rxjs/core';
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/hooks/useSelectedChain.ts:1]: we should add an unit test, even if it's trivial

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v3/src/hooks/useSelectedChain.ts (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useSelectedChain.ts:1]: Got it, I will read docs that you sent for stream testing and add tests :D

File: packages/tempus-client_v3/src/hooks/useSelectedChain.ts (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useSelectedChain.ts:4]: If we want to pipe other states to this one yes - for example, when chain changes, I want to update tokenBalances for example.

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**